vnext/Desktop/React.Windows.Desktop.vcxproj.filters

```diff
@@ -135,6 +133,13 @@
     <ClCompile Include="WebSocketResourceFactory.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
+    <ClCompile Include="..\Microsoft.ReactNative\IReactNotificationService.cpp" />
```

Comment:
FYI a bunch of these are 0.62 only. You may need to change this if backporting.

Reply:
Thanks for the notice. This is the filters file, though. Worst case, it would show unavailable files.
This was added automatically by Visual Studio.
The IDE will do the proper thing for backports that don't have these files as part of the VCXPROJ items.